src/repositories/cdn.js

```diff
@@ -40,6 +41,13 @@ class Cdn {
     return this.s3Client.upload(this.bucketName, sanitizedObjectName, stream, contentType, { ...defaultMetadata, ...metadata });
   }
 
+  async uploadObjectFromUrl(objectName, url) {
+    const body = await superagent.get(url).then(res => res.body);
+    const sanitizedObjectName = objectName.replace(/\\/g, '/');
```

Comment:
Should not the imported object have the exact same name as the source object? Also, in which case would we have a backslash as a separator? :thinking:

Reply:
True, I ran the same sanitizing as for `uploadObject` but `uploadObjectFromUrl` is used for import only and should keep the name as is, though from the perspective of the cdn module the reason for this difference is not clear.
Will remove anyway.

Reply:
Done!